### ISSUE TITLE
Not an issue, a request

### ISSUE THREAD

**Issue by bjd-pfq** [MaartenBaert/ssr#487]
Provide a way to specify a delay (say, in seconds), after the Start Recording button is pressed, before the recording actually commences (and maybe have some rundown animation, but that isn't important).

I find myself regularly recording stuff where I have to press the start button on the video, and simultaneously press the start button on the recorder, which in essence is impossible.

The delay will enable the user to achieve that simultaneity with some degree of accuracy.

**Comment by MaartenBaert** [comment]
This is already on the todo list (timer/scheduling). I don't think it would help much though. The only way to get this perfect is to edit your recording afterwards, you can never start at exactly the right frame.

Why are you trying to record a video that is already pre-recorded? Can't you just download it?

**Comment by bjd-pfq** [comment]
This one:

https://www.youtube.com/watch?v=1SgHUXUlCwI&list=PL0yYA9IlBHjBW44MIAyRwuJn4FLHMvwd0&index=174

is not handled by Videodownloadhelper.
So SSR was a bigger helper ;-)

Thanks.



MaartenBaert wrote on 09/10/2016 11:45 PM:

**Comment by MaartenBaert** [comment]
Try [youtube-dl](https://rg3.github.io/youtube-dl/).

**Comment by bjd-pfq** [comment]
True, I had youtube-dl installed for some time, though never used it.

Am now trying that very URL, but the odd thing is, it starts downloading
the /entire/ playlist (183 video's)...

I'll look into this, meanwhile, thanks for the tip.
Btw. SSR is magnificent, RMD cannot hold a candle to it. You should push
to get it installed as the default video grabber of say Linux Mint.

bjd



MaartenBaert wrote on 09/10/2016 11:51 PM:

**Comment by Kicer86** [comment]
@bjd-pfq you can start recording, then run video and finally remove frames from the beginning in some video editor.

**Comment by QuImUfu** [comment]
@bjd-pfq 
It will download the whole playlist, if "&list=PL0yYA9IlBHjBW44MIAyRwuJn4FLHMvwd0" is in the URL.
If you use "youtube-dl https://www.youtube.com/watch?v=1SgHUXUlCwI" it will download only that video.
I think this is a great feature.
